testgrid/config/config.yaml

```diff
@@ -1937,6 +1940,11 @@ dashboards:
   - name: gce-min-node-permissions
     test_group_name: ci-kubernetes-e2e-gce-min-node-permissions
 
+- name: google-gce-compute-image-tools
+  dashboard_tab:
+    - name: ci-daisy-e2e
```

Comment:
Would do 'daisy-e2e' or 'daisy' to be more in line with other names.

Reply:
Done.